plugins/filter_cgolib/Readme.md

```diff
@@ -0,0 +1,111 @@
+
+
+call golib from c
+
+example go src:
+
+```go
+
+package main
+
+import "C"
+import (
+	"fmt"
+)
+
+//export GoLibInit
+func GoLibInit(name, value []string) int {
```

Comment:
Is this function intended to be a no-op?

Reply:
This will init global parameters in go.  such as :  kubernetes has client-go for visit k8s's resources, can init in this func

Reply:
ok